spec/deck_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/deck'
+require './lib/card'
+
+RSpec.describe Deck do
+  let(:card_1) { Card.new("What is the capital of Alaska?", "Juneau", :Geography) }
+  let(:card_2) { Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM) }
+  let(:card_3) { Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM) }
```

Comment:
Looks good! Suggestion here:

When we find ourselves exceeding 80 characters per line, we should start thinking about breaking the line up into multi-lines :-)

So here, we can do:

```
let(:card_1) do
  Card.new("What is the capital of Alaska?", "Juneau", :Geography)
end
let(:card_2) do
  Card.new(
    "The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?",
    "Mars", 
    :STEM
  )
end
let(:card_3) do
  Card.new(
    "Describe in words the exact direction that is 697.5° clockwise from due north?", 
    "North north west", 
    :STEM
  )
end
```